compiler/ballerina-parser/src/test/resources/expressions/unary-expr/unary_expr_assert_13.bal

```diff
@@ -0,0 +1,6 @@
+function foo() {
+    boolean is\not\found = true;
```

Comment:
We use `\` to escape the next character. We only escape characters that are not allowed as the identifier following char. Here we can escape the whitespace. 
```suggestion
    boolean is\ not\ found = true;
```
`is\ not\ found` is equivalent to `is not found`. Since we can't parse as it is, we escape the special characters.